src/ceph-volume/ceph_volume/util/__init__.py

```diff
@@ -1,6 +1,7 @@
 import logging
 from math import floor
 from ceph_volume import terminal
+from six.moves import input
```

Comment:
ceph-volume doesn't explicitly require `python-six` (although Ceph does), this would require ceph-volume to declare the explicit new dependency in its setup.py files so that ad-hoc/unit tests work. I would prefer if you could solve this problem by doing a try/except as it is done elsewhere in the project to avoid using the `six` library

Reply:
IMHO this is bad coding style and not necessary because the six package is already a dependency of Ceph (e.g. the dashboard mgr module is using it).
But to keep the changes small i've adapted the code as wanted.